src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputEmptyBlocksAndCatchBlocks.java

```diff
@@ -7,23 +7,21 @@ class InputEmptyBlocksAndCatchBlocks {
   public void fooMethod() {
     InputEmptyBlocksAndCatchBlocks r = new InputEmptyBlocksAndCatchBlocks();
     int a = 1;
-    if (a == 1) { } // no warn 'Empty if block.' until #15338
+    if (a == 1) {}
```

Comment:
Let's keep violation here on such empty block with space.
It will keep proving that we violate such code.
Correct form of such code, without  space, we have in InputFormatted version of this file